services/vision/mlvision/classifier.go

```diff
@@ -165,3 +170,14 @@ func checkIfClassifierWorks(ctx context.Context, cf classification.Classifier) e
 	}
 	return nil
 }
+
+// createClassificationFilter creates a post processor function that filters on the outputs of the model.
+func createClassificationFilter(minConf float64, labelMap map[string]float64) classification.Postprocessor {
+	if len(labelMap) != 0 {
+		return classification.NewLabelConfidenceFilter(labelMap)
```

Comment:
Not sure I'm into the one or the other vibe for the filtering. It's also unclear which is prioritized.

Reply:
In the scope doc in the goals we said the filter confidence map would be prioritized, so I'm following the scope goals